### DISCUSSION TITLE
version info aufter update still on old level

### DISCUSSION THREAD

**Discussion by urmel1960** [kimai/kimai#2687] [On-Premise: Installation & Updates] [answered]
Hi there

I upgraded to verion 1.14.3 follwing the instructions in the documentation. 

After all it still says 1.14.1 in doctors page

Any suggestions ?

**Comment by kevinpapst** [comment, marked as answer]
I am happy that I have so many users that take care. Thanks 👍 

See #2688 and #2648 and #2648. 

Suggestion: ignore the version info. If you updated to the correct git tag, then everything is working as expected.

**Reply by urmel1960** [reply]
It would just be good to fix such things right away instead of having a whole bunch of other people unnecessarily running into the same problem. Those who have encountered this problem have only wanted to know their installation in a correct and proper state and rely on such obviously for you "non-essential things" like a correct versioning.

**Reply by kevinpapst** [reply]
You can always send a PR to fix this. Having expectations and blaming people who not only give you a free software, but also spent their only available lifetime to answer questions which were already answered multiple times before - well, gives you bad karma

**Reply by kevinpapst** [reply]
@urmel1960 Do you really think that [convincing others that Kimai is not the right choice for an](https://github.com/wekan/wekan/issues/812#issuecomment-888045106) [open source collaboration](https://github.com/wekan/wekan/issues/812) is the right way to react? I can only assume that you don't really understand the open source movement and how much of our lives the developers invest into it. Talking bad about the back of someone is never a good way to deal with communication issues, but this is really heart breaking. Sad to see postings like yours.
